test/test_force.js

```diff
@@ -165,9 +165,8 @@ function updatePluginRepo(tmpDir, os, pluginRepoDir, sdkBranch) {
 //
 // Create and compile app
 //
-function createCompileApp(tmpDir, os, appType, templateRepoUri, pluginRepoUri) {
```

Comment:
The caller (see https://github.com/wmathurin/SalesforceMobileSDK-Package/blob/857e44b3e80382c33a36d898029bbb5cfc375e24/test/test_force.js#L109) is figuring out the actual app type (even in the case of a template generation).